tests/conftest.py

```diff
@@ -16,9 +16,16 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 """Define all fixtures."""
 
+import json
+import pathlib
+
 import numpy as np
 import pytest
 
+DATA_FOLDER = pathlib.Path(__file__).parent / "data"
+PIR_TO_XY_FOLDER = DATA_FOLDER / "sync" / "pir_to_xy"
+PIR_TO_XY_RESPONSES = sorted(PIR_TO_XY_FOLDER.rglob("*json"))
```

Comment:
The `PIR_TO_XY_FOLDER` has only the response JSON files in it, right? Because reading `rglob("*json")` makes one think there might be subfolders and non-JSON files. Maybe `sorted(PIR_TO_XY_FOLDER.iterdir())` would also do the trick?

Reply:
Great point!

Reply:
Fixed in 4e978e1a6279dd41940993f361ab908a083d1796